README.md

```diff
@@ -65,6 +65,32 @@ NightfallDLP requires a config file and a few Environment variables in order to
 
 
 ```
```

Comment:
Let me know what you guys think about the location of this piece. I personally think that this extra info causes the `Action Spec` to be a little bloated, so open to moving it into its own section further down

Reply:
I personally think this text should appear before the example config file. And then we can include these config values in the complete config.json example